cli/tools/run/mod.rs

```diff
@@ -178,7 +178,19 @@ pub async fn eval_command(
   Ok(exit_code)
 }
 
+fn get_npm_config_user_agent() -> String {
+  format!(
+    "deno/{} npm/? deno/{} {} {}",
+    env!("CARGO_PKG_VERSION"),
+    env!("CARGO_PKG_VERSION"),
+    std::env::consts::OS,
+    std::env::consts::ARCH
+  )
+}
+
 pub async fn maybe_npm_install(factory: &CliFactory) -> Result<(), AnyError> {
+  std::env::set_var("npm_config_user_agent", get_npm_config_user_agent());
```

Comment:
This is not really correct. I believe we want to set it every time we install npm packages, when executing `deno run npm:<package_name>` and when running `deno task` that calls a bin entry of a package.